### DISCUSSION TITLE
Potential RAG use cases

### DISCUSSION THREAD

**Discussion by benWallace57** [nhsengland/ds_251_RAG#11] [Ideas]
A thread to think in a RAG way and notice projects that could benefit from the technique. 
Maybe it's worth tagging these with the benefits to the project. 

NHS conditions tool:
Database: NHS A-Z conditions
Metadata: Name (Maybe hyperlinked text could be captured, or "SEE A GP IF" boxes) 
Benefit: Flashy

DS Confluence Chat bot
Database: All confluence pages
Metadata: Page title, Dates of publications, updates, authors, owners and contributors (Conflunce export tool has the ability to add lots of metadata fields)
Benefit: DS interest

AI Standards Seacher
Database: All AI standards
Metadata: Standards agency, author, tags? I think standards are often tagged with topics 
Benefit: Real stakeholder interest

**Comment by SamHollings** [comment]
Code finder
Database: Github from relevant organisations, e.g. NHS england / Digital / X / DHSC / etc.
Metadata: repo slug, authoring organisation, any topics they've applied to the repo
Benefit: hopefully make our code more discoverable?
